project/SpytRelease.scala

```diff
@@ -70,7 +70,10 @@ object SpytRelease {
 
   private lazy val setCustomSpytVersions: ReleaseStep = {
     customSpytVersion.map { v =>
-      setVersionForced(Seq(spytVersion -> v, spytPythonVersion -> v), spytVersionFile)
+      setVersionForced(Seq(
+        spytVersion -> SpytSnapshot.SnapshotVersion.parse(v).toScalaString,
```

Comment:
This is not true, because just for this pattern I added this case https://github.com/ytsaurus/ytsaurus-spyt/blob/3c75d379a7ba41a1f75229bcb3503bf3560bdd8e/project/SpytSnapshot.scala#L31